app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -237,4 +334,14 @@ private boolean isFragmentAdded() {
     private FragmentActivity getActivity() {
         return parentFragment.getActivity();
     }
+
+    public void dispose() {
+        if (disposables != null && !disposables.isDisposed()) {
+            disposables.clear();
+        }
+    }
+
+    public String getLeadingImageEditLang() {
+        return isTranslation ? targetSummary.getPageTitle().getWikiSite().languageCode() : sourceSummary.getPageTitle().getWikiSite().languageCode();
+    }
```

Comment:
This will produce a NPE and crash the app if `sourceSummary` or `targetSummary` is null.  This can happen if the activity is destroyed (e.g. "don't keep activities) and then recreated when coming back from editing.